tdrs-frontend/package.json

```diff
@@ -66,7 +66,6 @@
   "devDependencies": {
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^11.2.6",
-    "@testing-library/user-event": "^7.1.2",
```

Comment:
This package was previously unused and wildly outdated, see also [this Dependabot PR](https://github.com/raft-tech/TANF-app/pull/919) for background on what prompted this change.